package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "carbon-state-management",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
I think this may already be at `0.0.2`, would a change not bump it to `0.0.3`?

Reply:
Yeah that would be the next step of releasing it surely?
Also it's only on `0.0.1` currently in master

Reply:
That's odd as it's on `0.0.2` on npm.

Reply:
If you're cutting a tag from this branch it should have the next version number in the `package.json`?

Reply:
Just realised there's no release branch so I suppose I can do it all in one haha